src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBean.java

```diff
@@ -150,10 +150,10 @@ public static ExternalTool parseAddExternalToolManifest(String manifest) {
         String scopeUserInput = getRequiredTopLevelField(jsonObject, SCOPE);
         String contentType = getOptionalTopLevelField(jsonObject, CONTENT_TYPE);
         //Legacy support - assume tool manifests without any mimetype are for tabular data
-        if(contentType==null) {
-            contentType=DataFileServiceBean.MIME_TYPE_TSV_ALT;
+        if (contentType == null) {
```

Comment:
Should this only be true for file scope tools?

Reply:
@scolapasta I don't have a strong opinion about this. I'll leave it alone unless you want me to change it.

Right now the docs reflect current state of the code (the "if" removed) in that I removed the statement in parenthesis highlighted in blue below.

![Screen Shot 2019-08-29 at 4 15 43 PM](https://user-images.githubusercontent.com/21006/63973197-4a1fe000-ca78-11e9-914b-8ad1872dd729.png)